tools/deny.toml

```diff
@@ -86,6 +83,9 @@ skip-tree = [
     # TestSys uses a newer version of base64 and serde_yaml
     { name = "testsys-model", version = "=0.0.8" },
     { name = "bottlerocket-types", version = "=0.0.8" },
+
+    # generate-readme uses an old version of clap
+    { name = "clap", version = "2.34.0" }
```

Comment:
Rather than skipping `clap`'s tree, I would just skip the tree for `generate-readme` itself.

Reply:
True, good call!